docs/the-top-10/introduction_for_ot.md

```diff
@@ -0,0 +1,29 @@
+# The security World for OT: Why is it problematic?
+
+Malicious external actors were previously not a problem for the operators.  These dangers were not relevant. In addition, direct physical access was usually required to manipulate the system.
+
+OT operators are familiar with the production process. Cybersecurity is a new field in which there is little knowledge.
```

Comment:
erm, maybe we should say that OT operators have little knowledge about cyber security? (;